rest/src/test/scala/io/udash/rest/RestTestApi.scala

```diff
@@ -94,6 +95,8 @@ case class ErrorWrapper[T](error: T)
 object ErrorWrapper extends HasPolyGenCodec[ErrorWrapper]
 
 trait RestTestApi {
+  final val neverGetCounter = Atomic(0)
```

Comment:
That's not an obvious way to do it and in production code would probably make this accessible via REST :D I'd move this to `impl()` and either:

1) Remove the explicit `RestTestApi` type from line 182 and at callsite - Scala will infer a more specific type that allows access to this field.
2) Add methods for getting the counter and resetting it without exposing the implementation.


also: add a comment if you can't think of a better name for this field, otherwise it's hard to understand why it's here without reading all callsites